pages/register/register.py

```diff
@@ -10,25 +10,28 @@
 @register_bp.route("/register", methods=["GET", "POST"])
 def register():
     form = RegisterForm()
+    reg_html = 'register.html'
     if form.validate_on_submit():
         print("FORM")
         username = form.username.data
         email = form.email.data.lower()
         password = form.password.data
 
+    
         if len(username) < 2:
             flash('Имя пользователя должно быть длиннее 1 символа', 'error')
-            return render_template('register.html', form=form)
+            return render_template(reg_html, form=form)
```

Comment:
As far as i know it's not best practice to define template string outside of the render_template function

Reply:
just for information